File: parallelization/R/load.R
#' Load TCGA breast cancer data
#'
#' @param path Path to the data
#'
#' @returns A tibble with the TCGA breast cancer data
load_brca_data <- function(path = here::here("data")) {
  brca_data <- readr::read_csv(
    file.path(path, "tcga_brca_array_data_filtered.csv")
  ) |>
    tibble::as_tibble()
  return(brca_data)
}


#' Load TCGA breast cancer subtype data
#'
#' @param path Path to the data
#'
#' @returns A tibble with the TCGA breast cancer subtype data
load_subtype_data <- function(path = here::here("data")) {
  subtype_data <- readr::read_csv(
    file.path(path, "tcga_brca_subtypes.csv")
  ) |>
    tibble::as_tibble()
  return(subtype_data)
}

File: parallelization/python/load.py
from os.path import join as oj
import pandas as pd


def load_brca_data(path='data'):
    '''
    Load TCGA breast cancer data
    
    Parameters
    ----------
    path : str
        Path to the data

    Returns
    -------
    pd.DataFrame
        A dataframe with the TCGA breast cancer data
    '''
    brca_data = pd.read_csv(oj(path, 'tcga_brca_array_data_filtered.csv'))
    return brca_data


def load_subtype_data(path='data'):
    '''
    Load TCGA breast cancer subtype data

    Parameters
    ----------
    path : str
        Path to the data

    Returns
    -------
    pd.DataFrame
        A dataframe with the TCGA breast cancer subtype data
    '''
    subtype_data = pd.read_csv(oj(path, 'tcga_brca_subtypes.csv'))
    return subtype_data
